src/evict/evict_lru.c

```diff
@@ -2353,6 +2353,11 @@ __wt_cache_eviction_worker(WT_SESSION_IMPL *session, bool busy, bool readonly, d
     txn_global = &conn->txn_global;
     txn_shared = WT_SESSION_TXN_SHARED(session);
 
+    if (session->cache_max_wait_us != 0)
```

Comment:
Is there a way to make a session wait infinitely when the connection cache_max_wait_us is finite?

Reply:
I don't understand this. If it is set to a finite value, then it will break out eviction if it times out.

Reply:
I mean, it seems there is no way I could configure '0' as cache_max_wait_us for a session, it will be overwritten by connection level cache_max_wait_us

Reply:
Yes, it is not possible. Do you have any idea to achieve this?